File: swipeback/src/main/java/com/aitangba/swipeback/SlideBackManager.java
package com.aitangba.swipeback;

import android.app.Activity;

public interface SlideBackManager {

    Activity getSlideActivity();

    /**
     * 是否支持滑动返回
     */
    boolean supportSlideBack();

    /**
     * 能否滑动返回至当前Activity
     */
    boolean canBeSlideBack();
}

File: swipeback/src/main/java/com/aitangba/swipeback/SlideBackManager.kt
package com.aitangba.swipeback

import android.app.Activity

interface SlideBackManager {

    fun getSlideActivity(): Activity?

    /**
     * 是否支持滑动返回
     */
    fun supportSlideBack(): Boolean

    /**
     * 能否滑动返回至当前Activity
     */
    fun canBeSlideBack(): Boolean

    fun getSwipeEdgeSize(): Int
}
